src/client/features/view/config.js

```diff
@@ -23,9 +23,16 @@ const databases = [
   ['CAS', 'http://identifiers.org/cas/', '']
 ];
 
+// Upper limit for mobile sizing in pixels, this is currently based off the iphone 6 plus
+const mobileUpperLimit = {
```

Comment:
Why is this needed?

Reply:
It's old code from before I switched search to `@media` I forgot to remove. Removed it.